### ISSUE TITLE
Can't bind keyboard events with modifiers

### ISSUE THREAD

**Issue by rdbende** [oooutlk/tcltk#5]
Keyboard events can't be bound with modifiers because the sequence text passed to tcl is malformed.

The book has this example:
> [...] and Command-period (`event::command().key_press( TkKey::period )`) to the close or cancel button.

To repro:
```rs
use tk::cmd::*;
use tk::*;

fn main() -> TkResult<()> {
    make_tk!()?
        .root()
        .add_ttk_button(())?
        .bind(event::control().key_press(TkKey::a), "destroy .")?;

    Ok(main_loop())
}
```

Error:
```
Error: InterpError(bad event type or keysym "Control<KeyPress"
    while executing
"bind .1 <Control<KeyPress-a> {destroy .}")
```

Fantastic project by the way! :)

**Comment by oooutlk** [comment]
I didn’t notice this issue. Will try to reproduce and fix in 12 hours.

**Comment by oooutlk** [comment]
Emm.. it should be `<Control-KeyPress-a>`.

**Comment by oooutlk** [comment]
Fixed in tk 0.1.6
